tests/wrapper_lake.py

```diff
@@ -4,14 +4,14 @@
 
 from lake.utils.util import generate_lake_config_wrapper
 from lake.utils.util import check_env
-from test_lake import base_lake_tester
+from lake.helper_test import base_lake_tester
```

Comment:
this will break right?

Reply:
yes, thanks for catching that! I'll update that.